src/components/modals/movieDetailsModal/movieDetailsModal.component.tsx

```diff
@@ -1,155 +1,129 @@
+import cN from 'classnames';
+import { Formik } from 'formik';
+import React, { FC } from 'react';
 import { useStoreon } from 'storeon/react';
-import React, { useState, FC } from 'react';
 import { StoreModule } from '@app/store/store.interface';
 import Button from '@app/components/button/button.component';
 import { IModal } from '@shared/interfaces/coreModal.interface';
 import { ButtonType } from '@app/components/button/button.interface';
 import { EMPTY_MOVIE, IMovie } from '@shared/interfaces/movies.model';
+import FormField from '@app/components/formField/formField.component';
 import MultipleDropdown from '@app/components/multipleDropdown/multipleDropdown.component';
-import useStyle from '@app/components/modals/movieDetailsModal/movieDetailsModal.component.style';
-import {
-  VALIDATION_DEFAULT_STATE,
-} from '@app/components/modals/movieDetailsModal/movieDetailsModal.interface';
+// eslint-disable max-len
+import validationSchema from '@app/components/modals/movieDetailsModal/movieDetailsModal.validation.schema';
+import { style as useStyle } from '@app/components/modals/movieDetailsModal/movieDetailsModal.component.style';
+// eslint-enable max-len
 
 const MovieDetailsModal: FC<IModal> = ({ onConfirmClick, onCancelClick, modalDetails }) => {
   const s = useStyle();
   const { movies } = useStoreon(StoreModule.movies);
-  const movieData = modalDetails.movie
-    ? movies.find((m: IMovie) => m.id === modalDetails.movie.id)
+  const movie = modalDetails.movie
+    ? { ...movies.find((m: IMovie) => m.id === modalDetails.movie.id) }
```

Comment:
does formik mutates `movie` object so you spread it here?

Reply:
Indeed, I make a shallow copy of movie when pass it into the Formik initial value, no need to do it twice.

> does formik mutates movie object so you spread it here?

Not sure, just wanted to make it a bit safer in case of any implicit mutations.